src/Classes/Utils/EventListeners.ts

```diff
@@ -116,36 +113,32 @@ export class EventListeners {
         const interactivityEl = container.interactivity.element;
 
         /* detect mouse pos - on hover / click event */
-        if (options.interactivity.events.onHover.enable || options.interactivity.events.onClick.enable) {
-            if (interactivityEl) {
-                /* el on mousemove */
-                interactivityEl.removeEventListener("mousemove", this.mouseMoveHandler);
+        if (interactivityEl && (options.interactivity.events.onHover.enable || options.interactivity.events.onClick.enable)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)